enhancements/hypershift/performanceprofilecontroller-node-tuning.md

```diff
@@ -381,7 +420,8 @@ In Hypershift Performance Profile controller (PPC) will not be reconciling Perfo
        - If it already exist
          - Extract the content and only if there is a difference update it properly and write it again.
    - Update PerformanceProfile status conditions with the status of the different components created from the PerformanceProfile (MachineConfig, KubeletConfig, Tuned)
-     - Still not defined if and how this will be done.
+     - PPC will create and update the performance profile status ConfigMap in the hosted control plane namespace, to reflect the updated status of the components. 
```

Comment:
PPC usually abbreviated as PerformanceProfileCreator which is not relevant here.